bigframes/pandas/__init__.py

```diff
@@ -679,6 +679,43 @@ def to_datetime(
 to_datetime.__doc__ = vendored_pandas_datetimes.to_datetime.__doc__
 
 
+def get_default_session_id() -> str:
+    """Gets the default session id. This is the session id that is
+    used whenever a custom session has not been provided. It is the
+    session id of the default global session. It is appended to the
+    table id of all temporary tables created in the global session.
+
+    Returns:
+        str, the default global session id, ex. 'sessiona1b2c'
+    """
+    return get_global_session().session_id
+
+
+def manual_cleanup_by_session_id(session_id: str) -> None:
```

Comment:
We don't need the word `manual` here. That's implied by the fact the user is calling it.

```suggestion
def cleanup_by_session_id(session_id: str) -> None:
```

Reply:
Ha, that's fair.